docs/architecture/adr/0027-angular-i18n-migration.md

```diff
@@ -0,0 +1,185 @@
+---
+adr: "0027"
+status: Proposed
+date: 2025-08-01
+tags: [clients, angular, i18n]
+---
+
+# 0027 - Improved Localization
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
+
+We currently use a homegrown localization system based on the
+https://developer.chrome.com/docs/extensions/reference/api/i18n api. This has served us reasonably
+well but comes with many limitations which we've had to workaround over the years.
+
+Most notable it lacks any native capability for interpolation which is useful for adding markdown
```

Comment:
```suggestion
Most notably it lacks any native capability for interpolation which is useful for adding markdown
```
🎨 I wouldn't make this a new paragraph since you're continuing the topic with "most notably".